Code/Tools/ProjectManager/Source/GemCatalog/GemModel.h

```diff
@@ -65,6 +65,7 @@ namespace O3DE::ProjectManager
         QVector<Tag> GetDependingGemTags(const QModelIndex& modelIndex);
         bool HasDependentGems(const QModelIndex& modelIndex) const;
 
+        static const GemInfo GetGemInfo(const QModelIndex& modelIndex);
```

Comment:
can you mark this function as const also?
```suggestion
        static const GemInfo GetGemInfo(const QModelIndex& modelIndex) const;
```

Reply:
Looks like I'm getting a compiler error stating that modifier is not allowed on static methods:

```
'GetGemInfo': modifiers not allowed on static member functions	
```

Reply:
Ah yeah good point, I didn't notice it was static.